roles/pulp/tasks/main.yaml

```
@@ -34,20 +115,43 @@
   ansible.builtin.systemd:
     daemon_reload: true
 
-- name: Start the Pulp Service
+- name: Start the Pulp API services
   ansible.builtin.systemd:
-    name: pulp
+    name: pulp-api
     enabled: true
-    state: restarted
+    state: started
+
+- name: Migrate the Pylp database
```

Eh, I typo'ed that :D
```suggestion
- name: Migrate the Pulp database
```